kobuki_dock_drive/include/kobuki_dock_drive/dock_drive.hpp

```diff
@@ -51,7 +51,7 @@
 #include <ecl/geometry/legacy_pose2d.hpp>
 #include <ecl/linear_algebra.hpp>
 
-#include "kobuki_dock_drive/state.hpp"
+#include "state.hpp"
```

Comment:
This seems like it will cause a build problem on linux. Please verify.

Reply:
verified on linux, this works since `#include "state.hpp"` first searches in the directory of this file

the reason why `#include "kobuki_dock_drive/state.hpp"` works:
`kobuki_dock_drive/include` is added to the includePath, so `kobuki_dock_drive/state.hpp` could be found under `kobuki_dock_drive/include`

but reverting this change since it falls into code cleanup and hence not necessary